android/app/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,350 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest
+  package="host.exp.exponent"
+  xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:tools="http://schemas.android.com/tools">
+
+  <permission
+    android:name="edu.haverford.MovementVitamins.permission.C2D_MESSAGE"
+    android:protectionLevel="signature"/>
+
+  <!-- These are required permissions to make the app run -->
+  <uses-permission android:name="edu.haverford.MovementVitamins.permission.C2D_MESSAGE" />
+  <uses-permission android:name="com.google.android.c2dm.permission.RECEIVE" />
+  <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
+  <uses-permission android:name="android.permission.INTERNET" />
+  <uses-permission android:name="android.permission.SYSTEM_ALERT_WINDOW" />
+  <uses-permission android:name="android.permission.WAKE_LOCK" />
+
+  <!-- ADD PERMISSIONS HERE -->
+  <!-- BEGIN OPTIONAL PERMISSIONS -->
+  <uses-permission android:name="android.permission.MANAGE_DOCUMENTS" />
+  <uses-permission android:name="android.permission.READ_INTERNAL_STORAGE" />
+  <uses-permission android:name="android.permission.READ_PHONE_STATE" />
+  <uses-permission android:name="android.permission.USE_FINGERPRINT" />
+  <uses-permission android:name="android.permission.VIBRATE" />
+  <uses-permission android:name="com.android.launcher.permission.INSTALL_SHORTCUT" />
+  <uses-permission android:name="android.permission.MODIFY_AUDIO_SETTINGS" />
+
+  <!-- These require runtime permissions on M -->
+  <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION" />
```

Comment:
Are we asking this user for all of these permissions? lack of knowledge coming into play here

Reply:
These were auto generated and we will fix them in a future update.